### DISCUSSION TITLE
createArrowFiles. <simpleError: No fragments found! >

### DISCUSSION THREAD

**Discussion by zahramesri** [GreenleafLab/ArchR#1402] [Questions / Documentation]
Hi. @rcorces , 

When I run: 
ArrowFiles <- createArrowFiles(
  inputFiles = inputFiles,
  minFrags = 1000,
  filterFrags = 1000,
  filterTSS = 4
)

I receive the following error and my ArrowFile is empty. 

2022-04-26 11:39:13 : ERROR Found in .tabixToTmp for (scATAC : 1 of 1) 
LogFile = ArchRLogs/ArchR-createArrows-12c042da2425-Date-2022-04-26_Time-11-38-45.log

<simpleError: No fragments found!>

I have attached the logFile here as well. Please let me know if anyone can help
[ArchR-createArrows-12c042da2425-Date-2022-04-26_Time-11-38-45.log](https://github.com/GreenleafLab/ArchR/files/8566155/ArchR-createArrows-12c042da2425-Date-2022-04-26_Time-11-38-45.log)

This issue also occurs on the example given in the tutorial.  
.

**Comment by rcorces** [comment]
1. please show the first 10 lines of your fragment file.
2.  Make sure that your fragment file is bgzipped, not gzipped

**Reply by zahramesri** [reply]
Thank you so much for your response. 
1.
chr1	10066	10192	GCGAGAAAGTGAAACT-1	1
chr1	10072	10339	TGGGTGCGTAAACGTA-5	1
chr1	10073	10192	CAATCCCGTTAAGGGC-2	5
chr1	10073	10267	TGGAAGGGTGATAGAT-5	1
chr1	10073	10284	TACAGCAAGACCCATT-2	1
chr1	10073	10315	ATGGATCCAAACGTTC-2	2
chr1	10073	10345	GCGATTATCTTGTACT-1	1
chr1	10078	10279	CACTAAGCAGGTGGTA-1	4
chr1	10078	10279	GCGTTGGAGCGATACG-5	3
chr1	10078	10309	TCACAAGCACCACAGC-1	1

2. my fragment file is .gz

**Reply by rcorces** [reply]
your fragments look fine. I know the file extension is `.gz` but files zipped with `bgzip` also have a `.gz` extension. Your other post made it sound like you had created these fragment files yourself. Is that true? Or is this from cell-ranger output?

**Reply by zahramesri** [reply]
I apologize for the confusion. The file is output from cell-ranger that I received from the original authors. Please let me know what other information I can provide. I really appreciate your assistance as this is my first time using ArchR and I am unsure of the error I am receiving following CreateArrowFile()

**Reply by rcorces** [reply]
can you clarify whether you are able to successfully create arrow files from the tutorial data?

**Reply by zahramesri** [reply]
No, the same error occurs when I try the tutorial data.

**Reply by rcorces** [reply]
can you try setting `threads = 1` in `createArrowFiles()`?

**Reply by zahramesri** [reply]
I tried on the tutorial example. I still get an error and I have attached the log file. 

ArrowFiles <- createArrowFiles(
  inputFiles = inputFiles,
  sampleNames = names(inputFiles),
  filterTSS = 4, #Dont set this too high because you can always increase later
  filterFrags = 1000, 
  addTileMat = TRUE,
  addGeneScoreMat = TRUE,
threads = 1
)

ERROR:

filterFrags is no longer a valid input. Please use minFrags! Setting filterFrags value to minFrags!
filterTSS is no longer a valid input. Please use minTSS! Setting filterTSS value to minTSS!
Using GeneAnnotation set by addArchRGenome(Hg19)!
Using GeneAnnotation set by addArchRGenome(Hg19)!
ArchR logging to : ArchRLogs/ArchR-createArrows-10d96779089e2-Date-2022-04-27_Time-16-21-09.log
If there is an issue, please report to github with logFile!
2022-04-27 16:21:09 : Batch Execution w/ safelapply!, 0 mins elapsed.
Attempting to index HemeFragments/scATAC_BMMC_R1.fragments.tsv.gz as tabix..
2022-04-27 16:21:09 : createArrowFiles has encountered an error, checking if any ArrowFiles completed..
ArchR logging successful to : ArchRLogs/ArchR-createArrows-10d96779089e2-Date-2022-04-27_Time-16-21-09.log
[ArchR-createArrows-10d96779089e2-Date-2022-04-27_Time-16-21-09.log](https://github.com/GreenleafLab/ArchR/files/8578396/ArchR-createArrows-10d96779089e2-Date-2022-04-27_Time-16-21-09.log)

**Reply by rcorces** [reply]
If you're getting an error on the tutorial data then that is a dead giveaway that something is likely wrong with your environment. Its going to be challenging for me to help troubleshoot that.
The only thing that might provide hints would be the output of `traceback()` after you receive this error on the tutorial data set.

**Reply by zahramesri** [reply]
Thank you so much Ryan for your help. 

I have tried and there is no luck there. 

> traceback()
No traceback available

**Reply by rcorces** [reply]
I'm sorry but without more information, there is no way for me to help you. You'll have to try to figure this out on your own.

**Reply by zahramesri** [reply]
Please let me know if there is any other information that I can share. Thank you again and I will try on my own.
